### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by PogChan**
Install and configure Cypress Component Testing
Created two simple test
Created a front end home page and an about page route

Installed Jest 
Installed SuperTest

### CODE REVIEW

File: client/jest.config.ts
Comment (wesleymui) [inline client/jest.config.ts:1]: Jest config file should be in the server folder i think

**Review by wesleymui — COMMENTED** [review]
Looks good, might need to move jest config file to server side since my workflow will be looking for it in server folder

**Review by wesleymui — APPROVED** [review]
lgtm

**Comment by wilgao09** [comment]
The server can be started with `npm run start -- (portnumber)` if you want to specify a port; if no port is supplied, it runs on 3031.

**Review by wilgao09 — APPROVED** [review]
lgtm

**Pull request merged**